lineman/app/components/thread_preview/thread_preview.haml

```diff
@@ -11,9 +11,8 @@
       %i.fa.fa-times>
       %span{translate: 'volume_levels.mute'}
 
-  %a.thread-preview__link{href: '/d/{{ thread.key }}'}
+  %a.thread-preview__link{lmo-href: '', lmo-href-model: 'thread'}
```

Comment:
Passing a string sometimes and a model other times is a pretty bad time I reckon, but this is a nice and clear compromise.

To be clear:
This:
```
a{lmo-href: 'dashboard'} # or '/dashboard'
```
will go to /dashboard, whereas this:
```
a{lmo-href: '', lmo-href-model: 'thread'}
```
Will figure out the path of the thread and go there.

If we find we need to pass query parameters at some point (we don't at the moment), we can add a `lmo-href-query` bit to handle that.